jbod_test.py

```
@@ -0,0 +1,125 @@
+        Check that all keys are on same disk.
+        """
+        cluster = self.cluster
+        cluster.populate(3)
+        node1, node2, node3 = cluster.nodelist()
+        cluster.set_configuration_options({'disk_failure_policy':'best_effort'})
+
+        cluster.set_data_dirs(['data1', 'data2'])
+        cluster.start()
+
+        cursor = self.patient_cql_connection(node1)
+        ksq = "CREATE KEYSPACE test WITH REPLICATION = {'class':'SimpleStrategy', 'replication_factor':3}"
+        cfq = "CREATE TABLE test.jtest(key int primary key, val int);"
+        cursor.execute(ksq)
+        cursor.execute(cfq)
+
+        #insert a bunch of keys
+        numkeys=100
+        for x in range(0, numkeys):
+            insq = "INSERT INTO test.jtest(key,val) VALUES ({key}, {key})".format(key=str(x))
+            cursor.execute(insq)
+        node1.flush()
+
+        initial_sstables = [None]*numkeys
+        #figure out which dir each key is in
+        for x in range(0, numkeys):
+            sstable_path = node1.nodetool('getsstables test jtest ' + str(x), capture_output=True)
+            if "data1" in sstable_path[0]:
+                initial_sstables[x] = "data1"
+            elif "data2" in sstable_path[0]:
+                initial_sstables[x] = "data2"
+            else:
+                self.fail(sstable_path)
+
+        #update added keys to ensure that keys remain on the same disk
+        for x in range(0, numkeys):
+            insq = "INSERT INTO test.jtest(key,val) VALUES ({key}, 66)".format(key=str(x))
+            cursor.execute(insq)
+        node1.flush()
+
+        #check that keys are on same disks as prior
+        final_sstables = [None]*numkeys
+        for x in range(0, numkeys):
+            sstable_paths = node1.nodetool('getsstables test jtest ' + str(x), capture_output=True)
+            for path in sstable_paths:
+                debug(initial_sstables[x])
+                debug(path)
+                if path != "":
```

Should we raise an error if the path is "". I don't think the test should pass if so.